### PR TITLE
Prepare for release 2.4.3

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

Depends-on: https://github.com/ansible/ansible-zuul-jobs/pull/1251

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -170,3 +170,11 @@ releases:
...
     - bugfix_100.yaml
     release_date: '2021-10-07'
+  2.4.3:
+    changes:
+      release_summary:
+        - Rereleased 2.4.2 with fix of network ee tests.
+    fragments:
+      - 2.4.3.yaml
+      - fix_network_ee.yaml
+    release_date: '2021-11-30'
```
Comment (rohitthakur2590) [inline changelogs/changelog.yaml:180]: we need to update this.

**Review by rohitthakur2590 — CHANGES_REQUESTED** [review]

File: changelogs/changelog.yaml (hunk 1 above)
Comment (ashwini-mhatre) [reply changelogs/changelog.yaml]: ack

**Review by rohitthakur2590 — APPROVED** [review]

**Pull request merged**